modules/oadp-usecase-restore.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-restore-different-namespace.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-usecase-restore_{context}"]
+= Restoring an application to a different namespace using {oadp-short}
+
+To restore a backup of an application by using {oadp-short}, you create a restore Custom Resource (CR) as shown in the following example. In the example, you restore the backup to a new target namespace, `test-restore-application`. The source namespace refers to the application namespace that you included in the backup. You then verify the restore by changing your project to the new restored namespace and verifying the resources.
```

Comment:
Usually, we just want one line above a procedure and that one line contains a "why" the user wants to do this; more than that ends up a concept module. Like, here, I might say something like, "Restore a backup of an application by using {oadp-short} to...." whatever the use case is.

Reply:
I will keep in this mind for my next PR :). Thank you.

Reply:
I have re-worded the intro as suggested by Shauna. But couldn't limit it to just one line. Since this procedure is a use case, I feel it makes sense to give the user an idea about what will be achieved. Thanks.